packages/serve-cli/src/run.ts

```diff
@@ -106,7 +105,14 @@ export async function run({
   version,
 }: RunOptions) {
   program = program.name(binName).description(productDescription);
-  if (version) program = program.version(version);
+  if (!version) {
+    version = await import('../package.json', { with: { type: 'json' } })
```

Comment:
@YassinEldeeb will it work once we have a binary?

Reply:
No it wont. We dont want to depend on the package.json, @ardatan.

Reply:
It wont work for docker either...

Reply:
it should be fine, local file imports like this would be embedded inline at the time of compilation to a binary target.

Reply:
but yeah it wouldn't work in docker as @enisdenjo mentioned, so I guess that should change.

Reply:
> I think it should be fine, local file imports like this would be embedded inline at the time of compilation to a binary target. but if it doesn't we can find a workaround later. I'm unsure

It wont be bundled, this is a dynamic import.

Reply:
Do you think we should minimize the differences between a binary and a docker image? I think so.

Maybe sharing a binary or at least a bundle could lead to less differences?

Reply:
> It wont be bundled, this is a dynamic import.

yeah, you're right